modules/statistics/php/statistics_site.class.inc

```diff
@@ -40,12 +40,20 @@ class Statistics_Site extends \NDB_Menu
      */
     function _hasAccess(\User $user) : bool
     {
-        $centerID = htmlspecialchars($_REQUEST['CenterID']);
         //TODO: Create a permission specific to statistics
-        $accessAllProfiles = $user->hasPermission('access_all_profiles') &&
-            $user->hasPermission('data_entry');
-        return $user->hasCenterPermission('data_entry', $centerID) ||
-            $accessAllProfiles;
+        $hasAccessToAllProfiles = $user->hasAllPermissions(
+            array(
+                'access_all_profiles',
+                'data_entry'
+            )
+        );
+        // If a CenterID is passed in the request, check if the user has the
+        // data_entry permission at the site/center specified by CenterID.
+        $hasCenterPermission = !empty($_REQUEST['CenterID'] ?? '') ?
```

Comment:
Using `$_REQUEST['CenterID'] ?? ''` fixes a PHP Notice when CenterID was not passed.